### PR TITLE
Add tcl example

### PR DESCRIPTION
**Pull request by kkelkar**
Added a `tcl` code and fixed a typo in `file_name` for `python_rockstar.py`

### CODE REVIEW

**Comment by avinassh** [comment]
Thanks for the PR! 
- please avoid using escape characters
- change the days to 400 from 777 

lastly, squash and merge 3 commits into one.

**Comment by kkelkar** [comment]
Made the changes. Can you have a look?

File: examples/tcl_rockstar.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from rockstar import RockStar
+
+tcl_code = """puts "Hello, World!"
+"""
```
Comment (avinassh) [inline examples/tcl_rockstar.py]: any reason for new line? I think following will be fine:

```
'puts "Hello, World!"'
```

**Comment by kkelkar** [comment]
Agreed, that can work as well. About the typo in `python_rockstar.py`, are you sure that change should be in the same commit?

**Comment by avinassh** [comment]
yup! Thanks, merged :)

**Pull request merged**